pkg/helmreconciler/listeners.go

```diff
@@ -444,9 +445,14 @@ func NewOwnerReferenceDecorator(instance runtime.Object) (RenderingListener, err
 	if err != nil {
 		return nil, err
 	}
+	gvk := schema.GroupVersionKind{
+		Version: "v1alpha2",
+		Group:   "install.istio.io",
+		Kind:    "IstioControlPlane",
+	}
```

Comment:
Since gvk seems used in a couple of places, it probably makes sense to define the whole thing as an exported const in a common package e.g. IstioOperatorGVK?

Reply:
put it into util/common.go